index.bs

```diff
@@ -704,6 +789,28 @@ signifies the end of the WebVTT cue.</p>
 
 </div>
 
+<div class="example">
+
+ <p>In this example, a sequence of video thumbnails and their text alternative are made available for the playback UI.</p>
+ <pre>
+WEBVTT
+
+ATTRIBUTES
+kind: metadata
+type: video-thumbnails
```

Comment:
@eric-carlson maybe this would be ready to merge if I removed this undefined `type` line for now. There is more to be done with `type` but we can update this example to re-include type once more specifics have been worked out for the type registry. What do you think?
```suggestion
```

Reply:
```suggestion

NOTE
The Timed Text Working Group is discussing a type registry for metadata 
`type` values, such as `video-thumbnails` or `video-flash-avoidance`. 
See webvtt issues #511 and #512 for more info.
```